lib/src/widgets/widgets.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: directives_ordering
```

Comment:
I want to sort it based on

adw
adw/new
gtk

But using this confuses adw and adw/new

Reply:
That's something else that I found a little confusing. I am not sure what the `new` folder is for. Why is `window_button_builder` in its own folder instead of in `adw`? Is it because it's not part of the standard libadwaita?

Anyway it's up to you but I would simply sort the import the standard way :)